libs/plpgsql.json

```diff
@@ -0,0 +1,46 @@
+  {"id": "plpgsql.pageinspect", "imports": ["pageinspect"], "name": "pageinspect", "repo": "postgres/postgres/tree/master/contrib/pageinspect", "tags": ["db-analysis"], "tech": ["eng-tools"]},
+  {"id": "plpgsql.passwordcheck", "imports": ["passwordcheck"], "name": "passwordcheck", "repo": "postgres/postgres/tree/master/contrib/passwordcheck", "tags": ["password-strength"], "tech": ["security"]},
+  {"id": "plpgsql.pg_buffercache", "imports": ["pg_buffercache"], "name": "pg_buffercache", "repo": "postgres/postgres/tree/master/contrib/pg_buffercache", "tags": ["db-analysis"], "tech": ["eng-tools"]},
+  {"id": "plpgsql.pgcrypto", "imports": ["pgcrypto"], "name": "pgcrypto", "repo": "postgres/postgres/tree/master/contrib/pgcrypto", "tags": ["security"], "tech": ["cryptography"]},
+  {"id": "plpgsql.pg-freespacemap", "imports": ["pg_freespacemap"], "name": "pg_freespacemap", "repo": "postgres/postgres/tree/master/contrib/pg_freespacemap", "tags": ["free-space-map"], "tech": ["eng-tools"]},
+  {"id": "plpgsql.pg-prewarm", "imports": ["pg_prewarm"], "name": "pg_prewarm", "repo": "postgres/postgres/tree/master/contrib/pg_prewarm", "tags": ["caching"], "tech": ["faster-code"]},
+  {"id": "plpgsql.pgrowlocks", "imports": ["pgrowlocks"], "name": "pgrowlocks", "repo": "postgres/postgres/tree/master/contrib/pgrowlocks", "tags": ["db-analysis"], "tech": ["eng-tools"]},
+  {"id": "plpgsql.pg-stat-statements", "imports": ["pg_stat_statements"], "name": "pg_stat_statements", "repo": "postgres/postgres/tree/master/contrib/pg_stat_statements", "tags": ["db-analysis"], "tech": ["eng-tools"]},
+  {"id": "plpgsql.pgstattuple", "imports": ["pgstattuple"], "name": "pgstattuple", "repo": "postgres/postgres/tree/master/contrib/pgstattuple", "tags": ["db-analysis"], "tech": ["eng-tools"]},
+  {"id": "plpgsql.pg-trgm", "imports": ["pg_trgm"], "name": "pg_trgm", "repo": "postgres/postgres/tree/master/contrib/pg_trgm", "tags": ["string-search"], "tech": ["faster-code"]},
+  {"id": "plpgsql.pg-visibility", "imports": ["pg_visibility"], "name": "pg_visibility", "repo": "postgres/postgres/tree/master/contrib/pg_visibility", "tags": ["db-analysis"], "tech": ["eng-tools"]},
+  {"id": "plpgsql.postgres-fdw", "imports": ["postgres_fdw"], "name": "postgres_fdw", "repo": "postgres/postgres/tree/master/contrib/postgres_fdw", "tags": ["db-link"], "tech": ["app-arch"]},
+  {"id": "plpgsql.seg", "imports": ["seg"], "name": "seg", "repo": "postgres/postgres/tree/master/contrib/seg", "tags": ["intervals"], "tech": ["data-structures"]},
+  {"id": "plpgsql.sepgsql", "imports": ["sepgsql"], "name": "sepgsql", "repo": "postgres/postgres/tree/master/contrib/sepgsql", "tags": ["access-control"], "tech": ["security"]},
+  {"id": "plpgsql.spi", "imports": ["spi"], "name": "spi", "repo": "postgres/postgres/tree/master/contrib/spi", "tags": ["spi"], "tech": ["api"]},
```

Comment:
tech == app-arch

api is for 3rd party apis. SPI helps build programs in a certain way (run SQL from C), so it's app-arch